public/content/roadmap/dencun/index.md

```diff
@@ -0,0 +1,151 @@
+- Lower gas costs make the network more accessible to new users and new use cases, which may reflexively increase demand for block space over time
+
+[EIP-7516](https://eips.ethereum.org/EIPS/eip-7516) - *Blob base fee opcode*
+
+- Introduces a new fee market, similar to the EIP-1559 mechanism, dedicated to pricing blob space
+- This helps offload demand for traditional L1 block space, reducing fees and reducing the amount of ETH burned on average
+
+### Deneb (Consensus) {#economic-impact-deneb}
+
+[EIP-4844](https://eips.ethereum.org/EIPS/eip-4844) - [See above](#economic-impact-cancun)
+
+[EIP-7044](https://eips.ethereum.org/EIPS/eip-7044) - *Perpetually valid signed voluntary exits*
+
+- This reduces the trust assumptions for users of pooled staking or staking-as-a-service platforms by providing a self-sovereign means of exiting validator funds without any permission from a third-party node operator
+- In the event this encourages more users to stake, the increase in total ETH staked will increase the issuance of new ETH
+- In the event users are more likely to exit staking, this will decrease the total ETH staked, reducing the issuance of new ETH
+
+[EIP-7514](https://eips.ethereum.org/EIPS/eip-7514) - *Add max epoch churn limit*
+
+- This caps the rate at which new ETH can be staked
+- ETH issuance is proportional to the total amount of ETH staked, thus this limitation will slow the potential issuance of new ETH
+- This does NOT mean that new validators cannot join, but removes the exponential climb in how many are allowed to join at a time
+
+## Will all transactions on L2s now use temporary blob space, or will you be able to choose? {#calldata-vs-blobs}
+
+Rollup transactions are not required to use blob space for their data (though they are economically incentivized to do so). Each can optionally write their data to blobs (guaranteed available for all challenge periods, very cheap) or to smart contract "calldata" (permanent storage, more expensive). Rollup providers will decide which to use based on the current demand for blob space. If blobs are in high demand, the rollup may decide to use calldata to ensure the data is posted in a timely manner.
+
+Although this decision is likely to be performed behind-the-scenes by rollup providers, it is theoretically possible for a user to choose which storage type to use. This would require the rollup provider to expose this option to the user, which introduces additional complexities when bundling transactions in a cost-effective manner. See individual rollup provider documentation for more details.
+
+## Would this reduce layer 1 (L1) gas fees in any way? {#l1-fee-impact}
+
+Yes, but significantly less than on layer 2 rollups. A new gas market is introduced exclusively for blob space, for use by rollup providers. This reduces the demand from these rollups for L1 block space, pushing gas prices downward.
```

Comment:
I think this should be phrased as more "no but maybe..."

- the upgrade does not affect the gas needed for an L1 transaction
- but L2 transaction do have an impact on gas of L1
- therefore if L2 transactions goes down, we might see a small reduction in fees

Reply:
An L2 that switches to using blob space is no longer consuming execution block space. This will reduce the burden for everyone else on L1.

Reply:
I just think we should find a more nuanced way of saying "yes... kinda" because it's only true (and to a hard to measure, potentially tiny amount?) if we're standing still.

> Yes, but significantly less than on layer 2 rollups.

what is significantly less?